### ISSUE TITLE
types of data augmenation

### ISSUE THREAD

**Issue by jsczzzk** [ucbdrive/hd3#9]
Thank you for your outstanding work! I have some problems about data augmentation.
I found that the model only uses scale,crop,HorizontalFlip and VerticalFlip to enchance the input data when you pretrain the model on chairs,not following the types of data augmentation in original [Flownet](https://github.com/NVlabs/PWC-Net/blob/864d2e9fb3f60f9af99f4ea72943b1ff5d472202/Caffe/model/PyCaffe/pwc_net_utils.py#L10) and only add RandomPhotometric augmentations for fine-tuning of MPI-Sintel. Will this implements lead to the overfit on MPI-Sintel test set? Will your results become better on MPI-Sintel test set if adopt data augmentations of Flownet?

**Comment by yzcjtr** [comment]
Hi @ZhongkaiZhou, thanks for your acknowledgment. We didn't notice the gain brought about by color augmentation on FlyingChairs before so we discarded it at last. It's hard to strictly follow the original practice of FlowNet in data augmentation for we adopt a distinctively different architecture here. It might help to try different augmentations but we didn't have time for that. Welcome to share your findings here.

**Comment by jsczzzk** [comment]
Thank you for your patient explation.
